<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>496</width>
    <height>302</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Prompt</string>
     </property>
     <property name="buddy">
      <cstring>promptLed</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="6">
    <widget class="QLineEdit" name="promptLed"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Negative Prompt</string>
     </property>
     <property name="buddy">
      <cstring>negativePromptLed</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="6">
    <widget class="QLineEdit" name="negativePromptLed"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Width</string>
     </property>
     <property name="buddy">
      <cstring>widthSbx</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QSpinBox" name="widthSbx">
     <property name="maximum">
      <number>8096</number>
     </property>
     <property name="value">
      <number>512</number>
     </property>
    </widget>
   </item>
   <item row="2" column="4">
    <widget class="QLabel" name="label_10">
     <property name="text">
      <string>Height</string>
     </property>
     <property name="buddy">
      <cstring>heightSbx</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="5" colspan="2">
    <widget class="QSpinBox" name="heightSbx">
     <property name="maximum">
      <number>8096</number>
     </property>
     <property name="value">
      <number>512</number>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Frames</string>
     </property>
     <property name="buddy">
      <cstring>framesSbx</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QSpinBox" name="framesSbx">
     <property name="toolTip">
      <string>Number of frames to generate. Only used when generating videos</string>
     </property>
    </widget>
   </item>
   <item row="3" column="4">
    <widget class="QLabel" name="label_11">
     <property name="text">
      <string>Length</string>
     </property>
     <property name="buddy">
      <cstring>lengthSbx</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="5" colspan="2">
    <widget class="QDoubleSpinBox" name="lengthSbx">
     <property name="toolTip">
      <string>Length in seconds of audio output</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Seed</string>
     </property>
     <property name="buddy">
      <cstring>seedLed</cstring>
     </property>
    </widget>
   </item>
   <item row="4" column="4" colspan="3">
    <widget class="QCheckBox" name="randomSeedCbx">
     <property name="text">
      <string>Random Seed</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>CFG Scale</string>
     </property>
     <property name="buddy">
      <cstring>cfgSbx</cstring>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QDoubleSpinBox" name="cfgSbx"/>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Steps</string>
     </property>
     <property name="buddy">
      <cstring>stepsSbx</cstring>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QSpinBox" name="stepsSbx"/>
   </item>
   <item row="6" column="4">
    <widget class="QLabel" name="label_12">
     <property name="text">
      <string>Sampler</string>
     </property>
     <property name="buddy">
      <cstring>samplerCmb</cstring>
     </property>
    </widget>
   </item>
   <item row="6" column="5" colspan="2">
    <widget class="QComboBox" name="samplerCmb"/>
   </item>
   <item row="7" column="0" colspan="3">
    <widget class="QCheckBox" name="inpaintingCbx">
     <property name="toolTip">
      <string>Enables inpainting sampling. Only available when sampling from an inpainting model</string>
     </property>
     <property name="text">
      <string>Inpainting</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>Base Image Path</string>
     </property>
     <property name="buddy">
      <cstring>imagePathLed</cstring>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="QLineEdit" name="imagePathLed">
     <property name="toolTip">
      <string>The base image used when inpainting</string>
     </property>
    </widget>
   </item>
   <item row="8" column="2">
    <widget class="QToolButton" name="imagePathBtn">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="8" column="3">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="8" column="4">
    <widget class="QLabel" name="label_9">
     <property name="text">
      <string>Mask Image Path</string>
     </property>
     <property name="buddy">
      <cstring>maskPathLed</cstring>
     </property>
    </widget>
   </item>
   <item row="8" column="5">
    <widget class="QLineEdit" name="maskPathLed">
     <property name="toolTip">
      <string>The mask used when inpainting</string>
     </property>
    </widget>
   </item>
   <item row="8" column="6">
    <widget class="QToolButton" name="maskPathBtn">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="seedLed"/>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>promptLed</tabstop>
  <tabstop>negativePromptLed</tabstop>
  <tabstop>widthSbx</tabstop>
  <tabstop>heightSbx</tabstop>
  <tabstop>framesSbx</tabstop>
  <tabstop>lengthSbx</tabstop>
  <tabstop>seedLed</tabstop>
  <tabstop>randomSeedCbx</tabstop>
  <tabstop>cfgSbx</tabstop>
  <tabstop>stepsSbx</tabstop>
  <tabstop>samplerCmb</tabstop>
  <tabstop>inpaintingCbx</tabstop>
  <tabstop>imagePathLed</tabstop>
  <tabstop>imagePathBtn</tabstop>
  <tabstop>maskPathLed</tabstop>
  <tabstop>maskPathBtn</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
